src/index.js

```diff
@@ -4,14 +4,17 @@ import "./index.css";
 import App from "./App";
 import { makeServer } from "./server";
 import { BrowserRouter } from "react-router-dom";
+import { FilterProvider } from "./froentend/contexts";
```

Comment:
Fix "frontend" typo in the folder structure

Reply:
yes fixed the Typo